### PR TITLE
Introduce metrics for messages sent and enforce consistency

### PR DESCRIPTION
**Pull request by pacospace**
## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

- Introduce metrics for messages sent so that we can see if the components are acting correctly
- Enforce consistency using Message().topic_name instead of using a string for the message (avoid errors)

### CODE REVIEW

File: kebechet_administrator.py [hunk 1]
```diff
@@ -88,7 +92,9 @@ def _handle_solved_message(Configuration):  # noqa: N803
         }
 
         # We store the message to put in the output file here.
-        output_messages.append({"topic_name": "thoth.kebechet-run-url-trigger", "message_contents": message_input})
+        output_messages.append(
+            {"topic_name": KebechetRunUrlTriggerMessage().topic_name, "message_contents": message_input}
```
Comment (KPostOffice) [inline kebechet_administrator.py:96]: do not put the full topic name. use `KebechetRunUrlTriggerMessage.base_name` the messaging cli will prepend the deployment name when *it* sends the message.

Comment (KPostOffice) [reply kebechet_administrator.py:96]: Here and elsewhere

**Review by KPostOffice — COMMENTED** [review]
One note otherwise I think it looks good

File: kebechet_administrator.py (hunk 1 above)
Comment (pacospace) [reply kebechet_administrator.py]: updated in 0807bf68c29b4228f30acdfdf1efad0816381c9a

**Review by KPostOffice — APPROVED** [review]
LGTM :+1:

**Pull request merged**